mod/developers/start.php

```diff
@@ -76,6 +76,16 @@ function developers_process_settings() {
 		$handler = [Hooks::class, 'alterMenu'];
 		elgg_register_plugin_hook_handler('view', 'navigation/menu/default', $handler);
 	}
+	
+	if (!empty($settings['block_email'])) {
```

Comment:
why check? it is also checked in the hooks right?

Reply:
copied from the rest